witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -193,7 +193,7 @@ getHashType someHash = fmap (fromMaybe HashIsUnknown) . runMaybeT . asum $
     isAddress = is
         (const HashIsAddress)
         addrFromText
-        (runSdReadMLocked @'ChainAndMempool . getMempoolAccountMaybe)
+        (\x -> runSdReadMLocked @'ChainAndMempool $ getMempoolAccountMaybe x)
```

Comment:
Why?

Reply:
`(.)` is not able of passing `WithinReadSDLock` constraint from `runSdReadMLocked` to `getMempoolAccountMaybe` :pepe_sad: